ci/code_checks.sh

```diff
@@ -126,6 +126,10 @@ if [[ -z "$CHECK" || "$CHECK" == "patterns" ]]; then
     invgrep -r -E --include '*.py' "[[:space:]] pytest.raises" pandas/tests/
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
+    MSG='Check for python2-style file encodings' ; echo $MSG
+    invgrep -R --include="*.py" -E "# -\*- coding: utf-8 -\*-" pandas scripts
```

Comment:
should this also explicitly include `.pyx` files, i.e. `--include=*.{py,pyx}`?

Reply:
Yes, nice find. Ive added that.